src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,170 @@
 package core.basesyntax;
 
+//import jdk.internal.util.ArraysSupport;
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
+
 public class ArrayList<T> implements List<T> {
+
+
+    private static final int def_capacity = 10;
+    private static final Object[] empty_arraydata = {};
```

Comment:
Wrong fromat. Ther have to be all uppercase letters.

[More info](https://mate-academy.github.io/style-guides/java/java.html#s7.1-javadoc-formatting:~:text=5.2.4-,Constant%20names,-Constant%20names%20use)

Also, this variable is unnecessary.